mlflow/utils/rest_utils.py

```diff
@@ -18,7 +18,8 @@
 }
 
 
-def http_request(host_creds, endpoint, retries=3, retry_interval=3, **kwargs):
+def http_request(host_creds, endpoint, retries=3, retry_interval=3,
+                 max_rate_limit_interval=60, **kwargs):
```

Comment:
Nit: update docstring for this method to mention the `max_rate_limit_interval` param? e.g. something like

```
 Makes an HTTP request with the specified method to the specified hostname/endpoint. Retries
    up to `retries` times if a request fails with a server error (e.g. error code 500), waiting
    `retry_interval` seconds between successive retries. Parses the API response (assumed to be
    JSON) into a Python object and returns it. If the request fails with a 429 (Too Many Requests), this method
    retries the request with exponential backoff until `max_rate_limit_interval` seconds have passed.
```

Reply:
Ah, I forgot to update the docstring, nice catch!